### ISSUE TITLE
Support for Razer Blade 15 2020 Advanced (300hz, 2080 SUPER MQ)

### ISSUE THREAD

**Issue by TheButlah** [openrazer/openrazer#1201]
Note, if this is a duplicate of #1159 , let me know and I will close this and move the device info there.

### Device Info

* Name: Not found under device. According to [here](https://support.razer.com/articles/1927452531) it is `Razer Blade 15" Advanced (2020)`
* Product Number: RZ09-03305E43
* Model Number: RZ09-0330

### Device Information
```
$ lsusb -d 1532: -v
Bus 001 Device 004: ID 1532:0253 Razer USA, Ltd Razer Blade
Couldn't open device, some information will be missing
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0 
  bDeviceSubClass         0 
  bDeviceProtocol         0 
  bMaxPacketSize0        64
  idVendor           0x1532 Razer USA, Ltd
  idProduct          0x0253 
  bcdDevice            2.00
  iManufacturer           1 
  iProduct                2 
  iSerial                 0 
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength       0x0054
    bNumInterfaces          3
    bConfigurationValue     1
    iConfiguration          0 
    bmAttributes         0xa0
      (Bus Powered)
      Remote Wakeup
    MaxPower              500mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      1 Boot Interface Subclass
      bInterfaceProtocol      1 Keyboard
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      61
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x81  EP 1 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        1
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0 
      bInterfaceProtocol      1 Keyboard
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength     159
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0010  1x 16 bytes
        bInterval               1
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        2
      bAlternateSetting       0
      bNumEndpoints           1
      bInterfaceClass         3 Human Interface Device
      bInterfaceSubClass      0 
      bInterfaceProtocol      2 Mouse
      iInterface              0 
        HID Device Descriptor:
          bLength                 9
          bDescriptorType        33
          bcdHID               1.11
          bCountryCode            0 Not supported
          bNumDescriptors         1
          bDescriptorType        34 Report
          wDescriptorLength      94
         Report Descriptors: 
           ** UNAVAILABLE **
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x83  EP 3 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0008  1x 8 bytes
        bInterval               1
```

```
$ ls -lah /dev/input/by-id/
total 0
drwxr-xr-x 2 root root 180 Jul 16 22:10 .
drwxr-xr-x 4 root root 680 Jul 16 22:10 ..
lrwxrwxrwx 1 root root   9 Jul 16 20:32 usb-Azurewave_Integrated_Camera_200901010001-event-if00 -> ../event4
lrwxrwxrwx 1 root root  10 Jul 16 20:32 usb-Azurewave_Integrated_Camera_200901010001-event-if02 -> ../event18
lrwxrwxrwx 1 root root  10 Jul 16 20:32 usb-Razer_Razer_Blade-event-if01 -> ../event14
lrwxrwxrwx 1 root root  10 Jul 16 20:32 usb-Razer_Razer_Blade-event-kbd -> ../event10
lrwxrwxrwx 1 root root  10 Jul 16 20:32 usb-Razer_Razer_Blade-if01-event-kbd -> ../event11
lrwxrwxrwx 1 root root  10 Jul 16 20:32 usb-Razer_Razer_Blade-if02-event-mouse -> ../event15
lrwxrwxrwx 1 root root   9 Jul 16 20:32 usb-Razer_Razer_Blade-if02-mouse -> ../mouse3
```

### Packet Captures

Captures me starting up Razer's Synapse software, and letting it run for a little bit: https://drive.google.com/file/d/1w2aSdeSqZjMzE4bZPiQJHnARePNA2vTe/view?usp=sharing

### Synapse Screenshots
<Screenshots of all available options as seen on Razer Synapse>

![image](https://user-images.githubusercontent.com/6969415/87746456-a1c55800-c7be-11ea-9249-fdbfee2bf044.png)

![image](https://user-images.githubusercontent.com/6969415/87743515-1e543880-c7b7-11ea-910f-7e6c9feec069.png)

![image](https://user-images.githubusercontent.com/6969415/87743542-30ce7200-c7b7-11ea-9900-c70d154a389e.png)

![image](https://user-images.githubusercontent.com/6969415/87743574-3cba3400-c7b7-11ea-935c-7990287cb371.png)

**Comment by z3ntu** [comment]
Yeah, it's a duplicate of #1159 as both devices have the same USB PID, but I'll keep it open as the PR already references it. You can also add a Fixes #1159 to the PR.

**Comment by TheButlah** [comment]
Update: I've edited my issue above to include a packet capture file
